### ISSUE TITLE
AG WG Review: 2.2.5 Re-authenticating (Level AAA) WCAG2ICT Guidance

### ISSUE THREAD

**Issue by pday1** [w3c/wcag2ict#899]
## Summary
Review of proposed WCAG2ICT guidance for the Level AAA WCAG SC: 2.2.5 Re-authenticating, as incorporated into the Editor's draft. 

## Maturity Level
Refining

## Action Needed
* Review content (See Content for Review section below for details.)
* Use the thumbs up and thumbs down icons to indicate agreement or disagreement with various comments. Do not add comments to simply support other comments. 
* Note any suggested edits for improvement and/or reasoning either in a comment on this issue or by creating a PR on the WCAG2ICT document.  
* Level A and AA SCs can be found in the [comments-by-guideline-and-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-by-guideline-and-success-criterion.md) file.
* Level AAA SCs can be found in the [comments-on-level-AAA-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-on-level-AAA-success-criteria.md)
----

## Background information 

Once finalized, this new WCAG2ICT Note will build on the previous version by also covering WCAG 2.2 level AAA success criteria (currently only level A and AA criteria are covered.  This is important as different jurisdictions are using different versions of WCAG for the accessibility of information and communication technology. 

## Content for Review

* Section [Applying SC 2.2.5 Re-authenticating to non-web documents and non-web software](https://w3c.github.io/wcag2ict/#applying-2-2-5-re-authenticating-to-non-web-documents-and-non-web-software).  

## Reference Material
In case you need them, links to various resources:

* [2.2.5 Re-authenticating](https://www.w3.org/TR/WCAG22/#re-authenticating) SC text in WCAG 2.2
* [Intent section of Understanding 2.2.5 Re-authenticating](https://www.w3.org/WAI/WCAG22/Understanding/re-authenticating#intent)

**Comment by maryjom** [comment]
Not particularly tied to the WCAG2ICT interpretation, but more on the WCAG side, when an authenticated session expires, there would have to be necessary safeguards for persistent data to avoid introducing security or privacy issues such as:
- not persisting sensitive data on the client side and/or ensuring it is encrypted
- ensuring any client side data is removed once users log out. 

IMO, WCAG should have an exception that applies if data retention would compromise security or process integrity.
In addition, there are some security standards requirements and HIPAA requirements that may need to be adhered to over saving private information, credit card information (PCI-DSS (Cardholder Data & SAD)), and protected health information, etc. that users may be providing as input. 

An article I found on this:
- [PCI Data Storage Do’s and Don’ts](https://www.pcisecuritystandards.org/pdfs/pci_fs_data_storage.pdf)

WCAG2ICT could include a note on this and, as a starting point, use the note found in the SC problematic for closed functionality section for 3.3.3 Accessible Authentication (Minimum) which probably should be elevated out of that section for use with this SC.
